api/tests/unit/domain/usecases/authentication/authenticate-oidc-user_test.js

```diff
@@ -34,11 +41,65 @@ describe('Unit | UseCase | authenticate-oidc-user', function () {
       };
 
       userRepository = { findByExternalIdentifier: sinon.stub() };
+      adminMemberRepository = {
+        get: sinon.stub(),
+      };
       userLoginRepository = {
         updateLastLoggedAt: sinon.stub().resolves(),
       };
     });
 
+    context('check access by audience', function () {
+      context('when audience is pix-admin', function () {
+        it('should throw an error when user has no role and is therefore not an admin member', async function () {
+          // given
+          const audience = appMessages.PIX_ADMIN.AUDIENCE;
+          _fakeOidcAPI({ oidcAuthenticationService, externalIdentityId });
+          userRepository.findByExternalIdentifier.resolves({ id: 10 });
+          adminMemberRepository.get.resolves(null);
+
+          // when
+          const error = await catchErr(authenticateOidcUser)({
+            audience,
+            oidcAuthenticationService,
+            userRepository,
+            adminMemberRepository,
+          });
+
+          // then
+          expect(error).to.be.an.instanceOf(ForbiddenAccess);
+          expect(error.message).to.be.equal('User does not have the rights to access the application');
+          expect(error.code).to.be.equal('PIX_ADMIN_ACCESS_NOT_ALLOWED');
+        });
+
+        it('should throw an error when user has a role but admin membership is disabled', async function () {
```

Comment:
```suggestion
        it('throws an error', async function () {
```

Il faudrait ajouter un contexte en plus avec `when user has a role but admin membership is disabled`